ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/ExecutionEngineChannelImpl.java

```diff
@@ -65,6 +66,9 @@ public static ExecutionEngineChannelImpl create(
       final Path beaconDataDirectory) {
     checkNotNull(eeEndpoint);
     checkNotNull(version);
+    if (eeEndpoint.equalsIgnoreCase(MOCK_ENDPOINT_IDENTIFIER)) {
+      return new ExecutionEngineChannelMock(spec, timeProvider, true);
```

Comment:
Can we log a really big warning here to the console please?

Reply:
SURE